package.json

```diff
@@ -104,5 +104,13 @@
   },
   "dependencies": {
     "patch-package": "^6.2.2"
+  },
+  "resolutions": {
+    "**/react-scripts/@typescript-eslint/parser": "^5.10.2",
+    "**/react-scripts/@typescript-eslint/eslint-plugin": "^5.10.2",
+    "**/react-scripts/eslint-plugin-jest": "^26.0.0",
+    "**/react-scripts/eslint-plugin-testing-library": "^5.0.1",
+    "**/react-scripts/eslint-config-react-app": "^7.0.0",
+    "**/eslint-config-react-app/eslint-plugin-jest": "^26.0.0"
```

Comment:
@turadg @michaelfig any qualms about this approach to avoid eslint plugin duplication?

Reply:
none that block it.

my thought though is that if we're going to control the versions at the root then to consider removing the dependencies from the package level. Some options:
- Rely on the root package for these
- Make a new package like `common` that has utilities used by many packages
- Expand the `eslint-config` package to be responsible for eslint deps too

Reply:
Unfortunately `resolutions` only work at the root with `yarn`. The problem is that we can't update `react-scripts` because of a breaking change, but it drags in `eslint` plugins as dependencies instead of `devDependencies`, and eslint plugins are notorious for causing issues when different versions lie around.

Reply:
Ah, I misread this as overriding all eslint, not just the `react-scripts` ones. No concerns at all.